bsp_diff/common/vendor/intel/external/project-celadon/cloud-streaming/0001-Add-entry-point-script-for-aic-cloud-env.patch

```diff
@@ -167,9 +168,14 @@ index 0000000..19edf25
 +
 +  local renderfps_enc_enable=${ENV_RENDERFPS_ENC_ENABLE:-"false"}
 +  local hevc_enable=${HEVC_ENABLE:-"false"}
++  local av1_enable=${AV1_ENABLE:-"false"}
```

Comment:
I think multiple `<codec>_enable` options are not convenient and just increase a code size. Can there be just one `codec` with avc being deault?
```
local codec=${CODEC:-"avc"}
if [[ "$codec" == "hevc" ]]; then
  irrv_codec="265"
elif [ "$codec" == "av1" ]; then
  irrv_codec="av1"
fi
```